packages/rs-dpp/src/state_transition/serialization.rs

```diff
@@ -88,6 +88,7 @@ mod tests {
         let identity_topup_transition = IdentityTopUpTransitionV0 {
```

Comment:
probably out of scope of this PR; but would be good to give IdentityTopUpTransitionV0 a default so you don't have to update so many places when there's a new field?

Reply:
Naw